### PR TITLE
add 11 Delete comments

### PR DESCRIPTION
**Pull request by YasirMaj**

### CODE REVIEW

File: src/components/Comments.jsx [hunk 1]
```diff
@@ -1,15 +1,36 @@
...
+    setComments((currComments) => {
+      return currComments.filter((comment) => {
+        return comment.comment_id !== commentToDelete.comment_id;
+      });
+    });
+    deleteComment(commentToDelete.comment_id).catch(() => {
+      setError("Something went wrong, please try again.");
+      setComments((currComments) => {
+        const newComments = [...currComments];
+        newComments.unshift(commentToDelete);
```
Comment (JVal-NC) [inline src/components/Comments.jsx]: This is ok but will disrupt the order of comments, could you instead save a copy of the original comments to preserve the order?

File: src/components/Comments.jsx [hunk 2]
```diff
@@ -1,15 +1,36 @@
...
+        return newComments;
+      });
+    });
+  };
 
   useEffect(() => {
     setIsLoading(true);
```
Comment (JVal-NC) [inline src/components/Comments.jsx:33]: Whats the purpose of this timeout?

File: src/components/Reviews.jsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  const [reviews, setReviews] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const { slug } = useParams();
+  const [order, setOrder] = useState("desc");
+  const [sort, setSort] = useState("reviews.created_at");
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  useEffect(() => {
+    setIsLoading(true);
+    setTimeout(() => {
```
Comment (JVal-NC) [inline src/components/Reviews.jsx:18]: Whats the purpose of the setTimeout?

**Review by JVal-NC — COMMENTED** [review]
See PR review comments

File: src/components/Reviews.jsx (hunk 3 above)
Comment (YasirMaj) [reply src/components/Reviews.jsx:18]: I was using it to test the loading icon, but I'll remove it

**Pull request merged**